Drakhtar/Scenes/Scene.cpp

```diff
@@ -1,3 +1,6 @@
+// Copyright 2019 the Drakhtar authors. All rights reserved. MIT license.
+
+#pragma once
```

Comment:
`#pragma once` in `.cpp` files are *virtual*ly useless, they're only for header files.

```suggestion
```